.github/workflows/build.yaml

```diff
@@ -63,9 +63,13 @@ jobs:
           mkdir -p /gcsfuse/wolfi-registry
           gcsfuse -o ro --implicit-dirs --only-dir os wolfi-production-registry-destination /gcsfuse/wolfi-registry
 
+          # Give gcsfuse some time to settle down.
+          sleep 30
+
           mkdir -p ./packages/${{ matrix.arch }}
+
           # Symlink the gcsfuse mount to ./packages/ to workaround the Makefile CWD assumptions
-          for f in /gcsfuse/wolfi-registry/${{ matrix.arch }}/*.apk; do
+          for f in $(find /gcsfuse/wolfi-registry/${{ matrix.arch }} -name '*.apk'); do
```

Comment:
would it be useful to specify non-zero size in find?

Reply:
It might be zero if nothing got built though iiuc